lib/methods/import.js

```diff
@@ -610,7 +648,9 @@ module.exports = self => {
             ];
           }, []);
 
-        if (duplicatedIds && relatedIds.every((id) => duplicatedIds.includes(id))) {
+        if (relatedIds.every((id) => duplicatedIds.has(id))) {
+          // TODO: Should we log an error here? I don't think so it's part of the natural process
+          // Maybe just a simple log
```

Comment:
isn't there a catch above that logs it?

Reply:
yes, one `insertOrUpdateAttachment` call logs the error and another one sets the job as a failure

Reply:
This isn't my point. We trigger an error while it's theoretically not an error.  If you son't import an attachment because the docs are duplicated it's not an error to me. Moreover errors are verbose in the terminal.. A simple log here would be enough to me?